app/DTO/WorkflowJobDTO.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DTO;
+
+class WorkflowJobDTO
+{
+    public function __construct(
+        public int $githubId,
+        public string $name,
+        public string $runner_os,
+        public string $runner_type,
```

Comment:
```suggestion
        public string $runnerOs,
        public string $runnerType,
```
please be consistent